### PR TITLE
Added Element rooms' links 

### PR DESCRIPTION
**Pull request by mhmohona**
Fixes -  #1677
Added info about element rooms for [Get in touch](https://conda-forge.org/docs/orga/getting-in-touch.html?highlight=gitter) page.

### CODE REVIEW

File: src/orga/getting-in-touch.rst [hunk 1]
```diff
@@ -26,6 +26,9 @@ The main chat rooms that you'll interact with are
 * `gitter: bot subteam <https://gitter.im/conda-forge/regro-cf-autotick-bot>`__: Public chat room for all things related to the conda-forge automation infrastructure.
   Our automation infrastructure is colloquially referred to as "the bot".
 * `gitter: compilers <https://gitter.im/conda-forge/conda-forge-compilers>`__: Public chat room focused on the conda-forge compiler stack.
+* `element: general <https://app.element.io/#/room/#conda-forge-space:matrix.org>`__: A general chat room for all things conda-forge on Element.
```
Comment (beckermr) [inline src/orga/getting-in-touch.rst:29]:
```suggestion
* `element: conda-forge-space <https://app.element.io/#/room/#conda-forge-space:matrix.org>`__: An element space listing all conda-forge rooms.
```

File: src/orga/getting-in-touch.rst [hunk 2]
```diff
@@ -26,6 +26,9 @@ The main chat rooms that you'll interact with are
...
+* `element: interns <https://app.element.io/#/room/#conda-forge_conda-forge-interns:gitter.im>`__: Public chat room for conda-forge interns to discuss projects and seek guidance from the community.
+* `element: community <https://app.element.io/#/room/#conda-forge:matrix.org>`__: Public chat room for the conda-forge community to discuss anything related to conda-forge.
```
Comment (beckermr) [inline src/orga/getting-in-touch.rst:31]:
```suggestion
* `element: general <https://app.element.io/#/room/#conda-forge:matrix.org>`__: Public chat room for the conda-forge community to discuss anything related to conda-forge.
```

**Review by beckermr — CHANGES_REQUESTED** [review]
Thanks for this PR! All gitter rooms are available from element now. They are actually one and the same. I think we'll need to adjust the PR to somehow reflect this.

File: src/orga/getting-in-touch.rst [hunk 3]
```diff
@@ -26,6 +26,9 @@ The main chat rooms that you'll interact with are
 * `gitter: bot subteam <https://gitter.im/conda-forge/regro-cf-autotick-bot>`__: Public chat room for all things related to the conda-forge automation infrastructure.
   Our automation infrastructure is colloquially referred to as "the bot".
 * `gitter: compilers <https://gitter.im/conda-forge/conda-forge-compilers>`__: Public chat room focused on the conda-forge compiler stack.
+* `element: conda-forge-space <https://app.element.io/#/room/#conda-forge-space:matrix.org>`__: An element space listing all conda-forge rooms.
+* `element: interns <https://app.element.io/#/room/#conda-forge_conda-forge-interns:gitter.im>`__: Public chat room for conda-forge interns to discuss projects and seek guidance from the community.
+* `element: general <https://app.element.io/#/room/#conda-forge:matrix.org>`__: Public chat room for the conda-forge community to discuss anything related to conda-forge.
```
Comment (jakirkham) [inline src/orga/getting-in-touch.rst:32]:
```suggestion

Note: All Gitter room links forward to Element

```

**Review by jakirkham — COMMENTED** [review]
Maybe something like this?

File: src/conf.py [hunk 4]
```diff
@@ -130,7 +130,8 @@
     r'https://polys.me/?$',  # 403 forbidden
     r'https://app.element.io/#/room/#conda-forge-space:matrix.org',  # needs login
     r'https://anacondacon.io/.*$',  # website is gone
-    r'https://matrix.to/#.*$',  # anchors are generated dynamically
+    r'https://matrix.io/#.*$',  # anchors are generated dynamically
```
Comment (beckermr) [inline src/conf.py:133]:
```suggestion
    r'https://matrix.to/#.*$',  # anchors are generated dynamically
```

**Review by beckermr — APPROVED** [review]

**Comment by jakirkham** [comment]
Thanks Mahfuza & Matt! 🙏

**Comment by jaimergp** [comment]
There are a few more instances of Gitter being mentioned which, I believe, should be replaced by Element/Matrix to avoid confusion.

See this [search query](https://github.com/search?q=repo%3Aconda-forge%2Fconda-forge.github.io+gitter+path%3A*.rst&type=code).

It might be worth considering opening a 2nd PR as a follow-up!

**Pull request merged**